api/filter/filter.go

```diff
@@ -332,13 +332,18 @@ func (f *Field) Expand() (expanded []Field) {
 // split field name.
 // format: resource.name
 // The resource may be "" (anonymous).
+// The (.) separator is escaped when preceded by (\).
 func (f *Field) split() (relation string, name string) {
 	s := f.Field.Value
 	mark := strings.Index(s, ".")
 	if mark == -1 {
 		name = s
 		return
 	}
+	if mark > 0 && s[mark-1] == '\\' {
+		name = s[:mark-1] + s[mark:]
+		return
+	}
```

Comment:
Support field names with (.) that are not nested resources (in the filter).
Needed to support GORM join queries which qualifies fields by table.
Example:
For a task query joined with application, all of the task fields are qualified as:
```
SELECT task.id, task.name ...
```
And application fields prefixed with `APPLICATION__`
```
SELECT APPLICATION__ID, APPLICATION__NAME ...
```